openapi3/schema_formats.go

```diff
@@ -35,4 +110,8 @@ func init() {
 
 	// date-time
 	DefineStringFormat("date-time", `^[0-9]{4}-(0[0-9]|10|11|12)-([0-2][0-9]|30|31)T[0-9]{2}:[0-9]{2}:[0-9]{2}(.[0-9]+)?(Z|(\+|-)[0-9]{2}:[0-9]{2})?$`)
+
+	DefineStringCallbackFormat("ipv4", validateIPv4)
+	DefineStringCallbackFormat("ipv6", validateIPv6)
```

Comment:
These are not part of the OAS so should not be predefined. cc https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#data-types

However I am not against
```go
func DefineIPv4Format() { ... }
func DefineIPv6Format() { ... }
```
so user can easily opt in.